openapi/kas-fleet-manager.yaml

```diff
@@ -899,6 +899,56 @@ paths:
       parameters:
         - $ref: "#/components/parameters/id"
   /api/kafkas_mgmt/v1/clusters:
+    get:
```

Comment:
might be good to add pagination and searching capabilities just like the list kafkas endpoint. But that may come in a follow up JIRA